### PR TITLE
PivotGridDataSource: update the remoteOperations property

### PR DESCRIPTION
**Pull request by sergepilipchuk**

### CODE REVIEW

File: api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/remoteOperations.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ If you enable this property, the PivotGrid sends several requests to load data.
         take: 20
     }
 
+[note]If you use [local stores](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/store/) in the PivotGridDataSource, disable the `remoteOperations` property to ensure that usage scenarios works properly.
```
Comment (vladaskorohodova) [inline api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/remoteOperations.md]: What stores are considered local? Provide examples

File: api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/remoteOperations.md [hunk 2]
```diff
@@ -56,6 +58,9 @@ Subsequent requests are different and contain the following **loadOptions**:
         ]
     }
 
+A sorting operation executes on a back-end if you enable the `remoteOperations` property and set the ['scrolling.mode'](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/scrolling/#mode) property to **'virtual'**
```
Comment (vladaskorohodova) [inline api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/remoteOperations.md]:
```suggestion
A sorting operation executes on a back-end if you enable the `remoteOperations` property and set the ['scrolling.mode'](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/scrolling/#mode) property to **'virtual'**.
```

File: api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/remoteOperations.md (hunk 1 above)
Comment (sergepilipchuk) [reply api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/remoteOperations.md]: ArrayStore and LocalStore

File: api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/remoteOperations.md [hunk 3]
```diff
@@ -56,6 +58,9 @@ Subsequent requests are different and contain the following **loadOptions**:
         ]
     }
 
+A sorting operation executes on a back-end if you enable the `remoteOperations` property and set the ['scrolling.mode'](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/scrolling/#mode) property to **'virtual'**.
+In all other cases, a sorting operation executes locally.
```
Comment (dirk-pieterse) [inline api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/remoteOperations.md]:
```suggestion
In all other cases, a sorting operation is executed locally. 
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**